tests/test_helpers.py

```diff
@@ -59,6 +74,23 @@ def test_add_extra_fields(centre_details):
 
         csv_to_test_reader = DictReader(fake_csv)
 
-        errors, augmented_data = add_extra_fields(csv_to_test_reader, centre_details[0], [])
+        errors, augmented_data = add_extra_fields(csv_to_test_reader, config.CENTRES[0], [])
         assert augmented_data == wrong_barcode
         assert len(errors) == 1
+
+
+def test_get_download_dir(config):
+    for centre in config.CENTRES:
+        assert get_download_dir(centre) == f"{DIR_DOWNLOADED_DATA}{centre['prefix']}/"
+
+
+def test_check_for_required_fields(config):
+    with pytest.raises(CentreFileError, match=r".* missing in CSV file"):
+        with StringIO() as fake_csv:
+            fake_csv.write("id,RNA ID\n")
+            fake_csv.write("1,RNA_0043_\n")
+            fake_csv.seek(0)
+
+            csv_to_test_reader = DictReader(fake_csv)
+
+            assert check_for_required_fields(csv_to_test_reader, {"barcode_field": "RNA ID"}) == []
```

Comment:
No tests for merge_daily_files itself? Is this because its tricky to direct the tests to use a dummy downloads directory?

Reply:
Nope, just didn't get round to it. Will try add some.